settings/DevHome.Settings/ViewModels/ExperimentalFeaturesViewModel.cs

```diff
@@ -40,7 +40,14 @@ private bool IsFeaturePresent(ExperimentalFeature experimentalFeature)
     {
         if (string.Equals(experimentalFeature.Id, "QuietBackgroundProcessesExperiment", StringComparison.OrdinalIgnoreCase))
         {
-            return QuietBackgroundProcessesSessionManager.IsFeaturePresent();
+            try
+            {
+                return QuietBackgroundProcessesSessionManager.IsFeaturePresent();
+            }
+            catch (System.Exception)

```

Comment:
Just to keep things clean let's take out the `System.` when we add the IsFeaturePresent check for the file explorer feature. Not worth changing here and running another pipeline.

Reply:
(yeah - i only added it because it gave me squigglies that i didn't understand :P)